### PR TITLE
Testing everything

### PR DESCRIPTION
**Pull request by ben-burlingham**
### Summary
Putting in a testing base on:
- Bulk Editor
- Date Select
- Filters
- List Partial
- Notification Bar
- Paginator
- Tiered Select
- Select
- Spinner
- Table
- Tabs
- Uploader

### Review Notes/ Questions/ TODOs
Basic "render" on all components. Using `yarn test` interface on all.  Test output report generating as well.

### Testing
Whoa, meta.

### CODE REVIEW

File: bulk-editor/test/BulkEditor.test.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  return Enzyme.shallow(
+    <BulkEditor {...mergedProps} />
+  );
+};
+
+describe('BulkEditor', () => {
+  /* TODO: Placeholder test, write better ones */
+  it('should render', () => {
+    const wrapper = shallowRender();
+    // expect(wrapper.exists()).toBe(true);
```
Comment (mcclayton) [inline bulk-editor/test/BulkEditor.test.jsx]: Why is the expectation commented out?

File: modal/.babelrc [hunk 2]
```diff
@@ -1,3 +1,6 @@
 {
   "presets": ["latest", "react", "stage-0"]
 }
+{
+  "presets": ["latest", "react", "stage-0"]
+}
```
Comment (mcclayton) [inline modal/.babelrc]: Did you have a script add these 😆 Duplicate

File: bulk-editor/test/BulkEditor.test.jsx (hunk 1 above)
Comment (ben-burlingham) [reply bulk-editor/test/BulkEditor.test.jsx]: Oh no! It's like that on all of them :/ Fixing.

File: modal/.babelrc (hunk 2 above)
Comment (ben-burlingham) [reply modal/.babelrc]: Fixed.

File: modal/package.json [hunk 3]
```diff
@@ -24,16 +30,18 @@
...
     "autoprefixer-loader": "^3.1.0",
     "babel-core": "^6.14.0",
+    "babel-jest": "^21.2.0",
     "babel-loader": "^6.2.5",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-plugin-transform-runtime": "^6.12.0",
     "babel-preset-latest": "^6.14.0",
     "babel-preset-react": "^6.11.1",
     "babel-preset-stage-0": "^6.5.0",
     "css-loader": "^0.21.0",
```
Comment (mcclayton) [inline modal/package.json]: Missing the dependency on the enzyme adapter here.

File: modal/README.md [hunk 4]
```diff
@@ -4,6 +4,9 @@ A Modal Component
 
 # Change Log
 
+##### 2.0.3
+- Add unit tests, CSS module mocking, and code coverage reporting.
```
Comment (mcclayton) [inline modal/README.md]: I think you are missing the Modal test file?

File: modal/package.json (hunk 3 above)
Comment (ben-burlingham) [reply modal/package.json]: Yep, Modal already had tests written and they're probably in Enzyme 2.  Didn't touch this one much hopefully, will revisit.

File: notification-bar/package.json [hunk 5]
```diff
@@ -20,19 +34,25 @@
   "homepage": "https://github.com/procore/particles/tree/master/notification-bar",
   "dependencies": {
     "classnames": "^2.2.5",
+    "enzyme-adapter-react-15": "^1.0.2",
```
Comment (mcclayton) [inline notification-bar/package.json]: All of these adapters should be able to be moved to the dev dependencies.

File: select/.gitignore [hunk 6]
```diff
@@ -1,3 +0,0 @@
-node_modules
```
Comment (mcclayton) [inline select/.gitignore]: Was there a reason this was removed?

File: bulk-editor/test/BulkEditor.test.jsx (hunk 1 above)
Comment (ben-burlingham) [reply bulk-editor/test/BulkEditor.test.jsx]: Fixed.

File: select/.gitignore (hunk 6 above)
Comment (ben-burlingham) [reply select/.gitignore]: Yep, the `.gitignore` at the root takes care of all of them.

File: spinner/package.json [hunk 7]
```diff
@@ -23,18 +37,24 @@
   "homepage": "https://github.com/procore/particles/tree/master/spinner",
   "dependencies": {
     "classnames": "^2.2.5",
+    "enzyme-adapter-react-15": "^1.0.2",
```
Comment (mcclayton) [inline spinner/package.json]: Wrong enzyme adapter version here. See the adapter chart: http://airbnb.io/enzyme/docs/guides/migration-from-2-to-3.html

File: table/.babelrc [hunk 8]
```diff
@@ -1,3 +1,6 @@
 {
   "presets": ["latest", "react", "stage-0"]
 }
+{
```
Comment (mcclayton) [inline table/.babelrc]: Duplicate

File: table/README.md [hunk 9]
```diff
@@ -12,6 +15,7 @@ A table component with self-managed checkboxes, sticky head, sticky right panel,
 - `tableData` prop is now `data`
 - `resizable` has been renamed to `userResizable` and has been removed from session storage
 - `default` object available in column definitions and `defaultWidth` was removed
+>>>>>>> 2a57210fb134004f7ff39351f0eb1eb7834b1926
```
Comment (mcclayton) [inline table/README.md]: From a merge conflict resolution?

File: spinner/package.json (hunk 7 above)
Comment (ben-burlingham) [reply spinner/package.json]: Good catch...not sure why tests still run 🤔  will update now

File: table/package.json [hunk 10]
```diff
@@ -32,16 +38,18 @@
   "devDependencies": {
     "autoprefixer-loader": "^3.2.0",
     "babel-core": "^6.25.0",
+    "babel-jest": "^21.2.0",
```
Comment (mcclayton) [inline table/package.json]: Missing enzyme adapter and missing Table.test

File: date-select/test/DateSelect.test.jsx [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const i18n = {
+};
+
+const shallowRender = (props = {}) => {
+  // Merge with defauls
```
Comment (dannyphillips) [inline date-select/test/DateSelect.test.jsx]: nb: typo

File: table/package.json (hunk 10 above)
Comment (ben-burlingham) [reply table/package.json]: Same as Modal - existing tests, I left it alone for now

File: table/README.md (hunk 9 above)
Comment (ben-burlingham) [reply table/README.md]: Perhaps? Fixing

File: select/package.json [hunk 12]
```diff
@@ -20,19 +34,25 @@
   "homepage": "https://github.com/procore/particles/tree/master/pickers/select",
   "dependencies": {
     "classnames": "^2.2.5",
+    "enzyme-adapter-react-15": "^1.0.2",
```
Comment (mcclayton) [inline select/package.json]: Wrong adapter version

File: tabs/.gitignore [hunk 13]
```diff
@@ -1,5 +0,0 @@
-node_modules
-.DS_STORE
-tmp
-npm-debug.log
```
Comment (dannyphillips) [inline tabs/.gitignore]: intentional?

File: modal/README.md (hunk 4 above)
Comment (ben-burlingham) [reply modal/README.md]: Already exists, see other comments

File: tabs/.gitignore (hunk 13 above)
Comment (ben-burlingham) [reply tabs/.gitignore]: Yep, .gitignore at the root takes care of all of them :)

File: date-select/test/DateSelect.test.jsx (hunk 11 above)
Comment (ben-burlingham) [reply date-select/test/DateSelect.test.jsx]: Fixed

File: spinner/package.json (hunk 7 above)
Comment (mcclayton) [reply spinner/package.json]: Probably would only surface in a more elaborate/complex test.

**Pull request merged**